chia/cmds/cmd_classes.py

```diff
@@ -301,3 +308,180 @@ async def wallet_rpc(self, **kwargs: Any) -> AsyncIterator[WalletClientInfo]:
                 config,
             ):
                 yield WalletClientInfo(wallet_client, fp, config)
+
+
+@command_helper
+class TransactionsIn:
+    transaction_file_in: str = option(
+        "--transaction-file-in",
```

Comment:
Again, this is an API change (there used to be a short `-i` option) but I think this will cause even less headache than the other API change.